core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -230,6 +233,41 @@ public ClientFactoryBuilder http2MaxFrameSize(int http2MaxFrameSize) {
         return this;
     }
 
+    /**
+     * Sets the maximum length of an HTTP/1 response initial line.
+     */
+    public ClientFactoryBuilder maxHttp1InitialLineLength(int maxHttp1InitialLineLength) {
+        checkArgument(maxHttp1InitialLineLength >= 0,
+                "maxHttp1InitialLineLength: %s (expected: >= 0)",
```

Comment:
nit: This line should be aligned like the below:
```
checkArgument(maxHttp1InitialLineLength >= 0,
              "maxHttp1InitialLineLength: %s (expected: >= 0)",
```
Would you check the code style section in [CONTRIBUTING.md](https://github.com/line/armeria/blob/master/CONTRIBUTING.md)?